contracts/governance/AxelarServiceGovernance.sol

```diff
@@ -15,20 +15,20 @@ contract AxelarServiceGovernance is InterchainGovernance, IAxelarServiceGovernan
         ScheduleTimeLockProposal,
         CancelTimeLockProposal,
         ApproveMultisigProposal,
```

Comment:
this as well (grep for all other multisig references)

Reply:
done.